### ISSUE TITLE
support bundled certs

### ISSUE THREAD

**Issue by homeyer** [waffleio/waffle.io-takeout#107]
It's valid in most systems for a certificate chain to be bundled (root and intermediate certs in one file). NodeJS does not support this out of the box, but we should, because it bites our customers.

**Acceptance Criteria**
- [ ] use https://github.com/bushong1/split-ca to parse any cert file we get
- [ ] verify it works for bundled and unbundled certs
- [ ] put this in helpers, bump helpers, and update helpers in every service

**Comment by jtslear** [comment]
@homeyer wouldn't this be the responsibility of Replicated?

**Comment by homeyer** [comment]
Nope, NodeJS doesn't handle bundled cert chains (multiple certs in one file). First search result I found: http://www.benjiegillam.com/2012/06/node-dot-js-ssl-certificate-chain/.

From the replicated side this is still just uploading one file, we just have to parse it correctly.

**Comment by homeyer** [comment]
The code to change is here: https://github.com/waffleio/waffle.io-helpers/blob/master/src/certs.coffee

**Comment by homeyer** [comment]
We’re doing some spring cleaning over at Waffle! 🌻

That means we’re closing out some issues we’re not committed to doing in the next 6 months or so, but please feel free to keep commenting on it, we’re still listening! More on our blog: https://blog.waffle.io/managing-large-backlogs-48db59425208
